From 3dd6893ea7069ceee9cc527a7286b3f01970ddcd Mon Sep 17 00:00:00 2001
From: "Friedrich W. H. Kossebau" <kossebau@kde.org>
Date: Wed, 8 Jul 2020 15:46:47 +0200
Subject: [PATCH 17/22] Find also Python3 with find_package(PythonInterp)

At least with openSUSE TW searching with a version given seems to restrict
the versions accepted to the major version.
So if only python3 is available, searching for 2.6 yields no results.
Given 2.6 is stone-age old, we can assume any found version is good enough
these days.
---
 cmake/modules/KDbCreateSharedDataClasses.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/KDbCreateSharedDataClasses.cmake b/cmake/modules/KDbCreateSharedDataClasses.cmake
index dc5101d9..b764cb36 100644
--- a/cmake/modules/KDbCreateSharedDataClasses.cmake
+++ b/cmake/modules/KDbCreateSharedDataClasses.cmake
@@ -23,7 +23,7 @@ macro(KDB_CREATE_SHARED_DATA_CLASSES)
     list(GET _args 1 PREFIX)
     list(REMOVE_AT _args 0 1)
     # message(STATUS "OUTPUT_VAR: ${OUTPUT_VAR} ${_args}")
-    find_package(PythonInterp 2.6)
+    find_package(PythonInterp)
     set_package_properties(PythonInterp PROPERTIES DESCRIPTION "Python language interpreter"
                            URL "https://www.python.org" TYPE REQUIRED
                            PURPOSE "Required by the Shared Data Classes (SDC) tool")
-- 
2.31.1

